packages/graphql/src/modules/user/app/authenticateFacebookUser.ts

```diff
@@ -47,43 +47,37 @@ const authenticateFacebookUser = ({
   const existingUser = users && users[0];
 
   if (existingUser) {
+    let updatedUser = existingUser;
```

Comment:
You don't need to create this var. Just call update function passing a new object: `await userRepository.update({...existingUser, name});`

Reply:
How would I return the updated user? Do you suggest querying it from the database? I

Reply:
Hum sorry, I didn't notice that you're using this var in other routines after update the user. Let it be.